src/compat/eme/set_media_keys.ts

```diff
@@ -43,27 +51,11 @@ function _setMediaKeys(
     return elt.setMediaKeys(mediaKeys);
   }
 
-  if (mediaKeys === null) {
-    return;
-  }
-
-  if ((elt as any).WebkitSetMediaKeys) {
-    /* tslint:disable no-unsafe-any */
-    return (elt as any).WebkitSetMediaKeys(mediaKeys);
-    /* tslint:enable no-unsafe-any */
-  }
-
   if ((elt as any).mozSetMediaKeys) {
     /* tslint:disable no-unsafe-any */
     return (elt as any).mozSetMediaKeys(mediaKeys);
     /* tslint:enable no-unsafe-any */
   }
-
-  if ((elt as any).msSetMediaKeys) {
-    /* tslint:disable no-unsafe-any */
-    return (elt as any).msSetMediaKeys(mediaKeys);
-    /* tslint:enable no-unsafe-any */
-  }
```

Comment:
Thinking about it, it does make more sense to use the same `_setVideo` callback in both removed cases.

BUT... Are we sure we don't break any case here? Isn't there cases where `msSetMediaKeys` is defined but where we don't follow the IE11MediaKeys variation?

Same question for WebkitSetMediaKeys.

Maybe we should keep both to be safe

Reply:
I'm a bit uncomfortable with this idea mainly for one reason. If we decide to use one of MS of WebKit API without going in the compatibility code, there are high chances that the code would crash precisely by the lack of compatibility (non-existing APIs).

Actually, i would find it safer to decide to be compatible just by the presence of the API.
For exemple, for WebKitMediaKeys, there are much more chances that the API would go into the same behavior between a Safari and another webkit based browser, no ?